### PR TITLE
Minor changes in README.md

### PR DESCRIPTION
**Pull request by jcpetruzza**
Main changes:

- More consistently using `Marge-bot` instead of `Marge`.
- Better explain the Access Token thing.
- Add a Troubleshooting section, mention the common issue with git hooks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,19 +33,21 @@ Since she is at it, she can optionally provide some other goodies like tagging o
...
 
-First, create a user for Marge on your GitLab. We'll use `marge-bot` here; but
-*we recommend you start the user name with a space*, e.g. ` marge-bot`, so Marge
-sorts topmost and is quick to assign to (our code strips trailing whitespace in
-the user name, so it won't show up elsewhere). Then add it to your projects as a
-developer or admin. For certain features, namely, `--impersonate-approvers`, and
-`--add-reviewed-by`, you will need to grant `marge-bot` admin privileges.
+First, create a user for Marge-bot on your GitLab. We'll use `marge-bot` as username here as well.
+GitLab sorts users by Name, so we recommend you pick one that starts with a space,
+e.g. ` Marge Bot`, so it is quicker to assign to (our code strips trailing whitespace in
```
Comment (aschmolck) [inline README.md]: ` Marge-bot` vs ` Marge Bot`?

Comment (jcpetruzza) [inline README.md]: Well, one thing is the the `username`, another thing is the user `name`. I could go with `Marge` for the user `name` to make the distinction more clear

Comment (aschmolck) [inline README.md]: I actually prefer "Marge Bot" to "Marge" as user name. We should change it in our installs if we go with this.

**Comment by aschmolck** [comment]
Looks good - but get an editor that can wrap lines properly:P

**Pull request merged**